Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/InternetExplorer/InternetExplorerConfiguration.cs

```diff
@@ -63,7 +56,7 @@ public override IBrowserFactory BrowserFactory
 
     public override IDownloadHelper DownloadHelper
     {
-      get { return _downloadHelper; }
+      get { return new NullDownloadHelper(); }

```

Comment:
nullobject or notsupportedexception? for the tests, the exception might be easier to discover than a nullobject?

Reply:
In `WebTestHelper.OnTearDown` we have this:

```csharp
_browserConfiguration.DownloadHelper.DeleteFiles();
```

Which is why I went with the "Do nothing" approach as oppose to do a "If not IE then delete" in `WebTestHelper`

Reply:
i.c.